requirements.txt

```diff
@@ -8,3 +8,4 @@ socketIO-client == 0.5.6
 stopit >= 1.1.1, < 1.1.999
 yamlconf >= 0.0.3, < 0.0.999
 revscoring >= 0.7.10
+redis
```

Comment:
Made it optional again in https://github.com/wiki-ai/ores/commit/5d135f848d0c3f084a7ffe3d67249a6d57f18ca2